agent/comms.go

```diff
@@ -119,29 +121,30 @@ func (a *orbAgent) startComms(config config.MQTTConfig) error {
 	return nil
 }
 
-func (a *orbAgent) subscribeGroupChannels(groups []fleet.GroupMembershipData) []string {
-	var successList []string
+func (a *orbAgent) subscribeGroupChannels(groups []fleet.GroupMembershipData) {
 	for _, groupData := range groups {
 
 		base := fmt.Sprintf("channels/%s/messages", groupData.ChannelID)
 		rpcFromCoreTopic := fmt.Sprintf("%s/%s", base, fleet.RPCFromCoreTopic)
 
 		token := a.client.Subscribe(rpcFromCoreTopic, 1, a.handleGroupRPCFromCore)
 		if token.Error() != nil {
-			a.logger.Error("failed to subscribe to group channel/topic", zap.String("topic", rpcFromCoreTopic), zap.Error(token.Error()))
+			a.logger.Error("failed to subscribe to group channel/topic", zap.String("name", groupData.Name), zap.String("topic", rpcFromCoreTopic), zap.Error(token.Error()))
```

Comment:
consistency: let's keep the key "group_name". also we might as well add "group_id" so we can search easier in the logs.